src/server/src/get-readable-path.ts

```diff
@@ -1,6 +1,13 @@
 import { homedir } from 'node:os'
 
-const home = homedir()
+let home: string
+try {
+  home = homedir()
+} catch {
+  // In restricted environments (like locked-down Codespaces),
+  // homedir() might fail. Fall back to current working directory.
+  home = process.cwd()
+}
```

Comment:
this is not necessary in the case we default to an empty array as suggested above